interpreter/exec/v128.ml

```diff
@@ -0,0 +1,5 @@
+include Vector.Make
```

Comment:
Maybe we can have v128 without using Vector? Having a Vector doesn't seem necessary at this point since we only have v128, unlike floats or ints.

Reply:
Yep, maybe. I thought about this too, but decided to still do it because this structure is assumed in some other places and operations on vectors probably would be relatively universal (as they would probably work on an array of bytes). Also, this might help with vectors 2.0 (whichever way they would end up).

Reply:
from a design standpoint, we should consider easy extension to >128bit types as well.